pages/resources/user.mdx

```diff
@@ -25,15 +25,42 @@ The following restrictions are additionally enforced for webhook names:
 
 There are other rules and restrictions not shared here for the sake of spam and abuse mitigation, but the majority of users won't encounter them. It's important to properly handle all error messages returned by Discord when editing or updating names.
 
+### Unique Usernames
+
+Discord's username system is changing. Discriminators are being removed and new, unique usernames (`@name`) and display names are being introduced. Internally, this migration is referred to as pomelo. You can read more details about how the changes to the username system affect user accounts in the [general Help Center article](https://dis.gd/usernames). To learn how it impacts bots specifically, you can read the [Developer Help Center article](https://dis.gd/app-usernames).
+
+A user's legacy `username#discriminator` will still be usable to [send friend requests](#send-friend-request), and will be available as a [profile badge](#profile-badge-structure) for migrated users.
+
+#### Identifying Migrated Users
+
+Pomelo will rollout to users over time rather than all at once. The value of a single zero (`0`) in the [`discriminator` field on the user object](#user-object) indicates that the user has ~~been pommeled~~ migrated to the new username system. Note that the discriminator for migrated users will _not_ be 4-digits like a standard discriminator (it is `0`, not `0000`). The value of the `username` field will become the migrated user's unique username.
```

Comment:
```suggestion
Pomelo will rollout to users over time rather than all at once. The value of a single zero (`0`) in the [`discriminator` field on the user object](#user-object) indicates that the user has ~~been pommelled~~ migrated to the new username system. Note that the discriminator for migrated users will _not_ be 4-digits like a standard discriminator (it is `0`, not `0000`). The value of the `username` field will become the migrated user's unique username.
```